go/mysql/query.go

```diff
@@ -416,7 +416,7 @@ func (c *Conn) ReadQueryResult(maxrows int, wantfields bool) (*sqltypes.Result,
 	for {
 		data, err := c.readEphemeralPacket()
 		if err != nil {
-			return nil, false, 0, err
+			return nil, false, 0, NewSQLError(CRServerLost, SSUnknownSQLState, "%v", err)
```

Comment:
Just wondering, why are we returning a `SSUnknownSQLState`?

Reply:
This is what MySQL set for SSState when CRServerLost is the error code.

Reply:
Yep exactly